documentation/modules/configuring/con-config-mirrormaker2-high-volume-messages.adoc

```diff
@@ -78,7 +78,7 @@ spec:
   - sourceCluster: "my-cluster-source"
     targetCluster: "my-cluster-target"
     sourceConnector:
-      tasksMax: 10
+      tasksMax: 2
```

Comment:
That's ok, but I would also set `spec.replicas: 1` instead of 5 to keep the example with minimal resources.

I think it is fine because we have notes that suggest to increase these values according to available resources.